example_payloads/variant_calling.payload.json

```diff
@@ -24,11 +24,10 @@
     {
```

Comment:
Just a general/curious question about this sample payload JSON --> Why is this file needed if @edsu7 is creating the actual payloads we will use?  Or is this just a reference for people who want to create payloads and need an example?

Reply:
Yes. Your understanding is absolutely correct. The payload under `example_payloads` is to give an example and a reference for people who want to create payloads. And also we can use it to validate with the schema for some testing purpose.